barretenberg/cpp/src/barretenberg/vm/avm/generated/flavor.hpp

```diff
@@ -308,7 +308,33 @@ class AvmFlavor {
         auto get_to_be_shifted() { return AvmFlavor::get_to_be_shifted<DataType>(*this); }
     };
 
-    using VerificationKey = VerificationKey_<PrecomputedEntities<Commitment>, VerifierCommitmentKey>;
+    // Note(md): required for instantiation from the proving key - im sure there are other ways to construct this
+    class VerificationKey : public VerificationKey_<PrecomputedEntities<Commitment>, VerifierCommitmentKey> {
+      public:
+        VerificationKey() = default;
+        VerificationKey(const size_t circuit_size, const size_t num_public_inputs)
+            : VerificationKey_(circuit_size, num_public_inputs)
+        {}
```

Comment:
Were these ones needed? Where are they used? (since they would in principle create an invalid VK)

Reply:
Good catch. I will remove it.